package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "muport-did-resolver",
-  "version": "0.1.3",
+  "version": "0.2.0-beta-4",
```

Comment:
Why the `beta-4` stuff?

Reply:
I needed to release a beta version so I could use it in the muport-core library when testing. Will make a release when this is merged.

Reply:
You'll make another PR removing the beta?

Reply:
I'll just do it in master. We don't have a develop branch yet, but if development on this repo get's more active I'll add one. For now I feel it's just a waste of time :)